### DISCUSSION TITLE
Third Party GraphQL API

### DISCUSSION THREAD

**Discussion by uniskela** [wp-graphql/wp-graphql#2346] [General]
Hey guys,

Please excuse my lack of knowledge and understanding of WPGraphQL and most of code development in general.

Is it possible to query a third party GraphQL database in WPGraphQL?

**Comment by justlevine** [comment]
Not sure I understand the question. Your frontend app should be able to query any graphql endpoint, the same way you query `yoursite.com/graphql`
